### PR TITLE
User can offer help

### PR DESCRIPTION
**Pull request by jysmys**
### User can offer to help
[PT link](https://www.pivotaltracker.com/story/show/173337447)
```
As a user,
In order to earn karma points,
I would like to be able to let the requester know I am interested in helping
```

- OfferMessage component created 
- Post action is created to /offers
- Sad path test is already covered in "visitorCanViewSpecificRequest.feature"

image

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -207,19 +228,30 @@ textarea {
...
+}
+#send-message-form {
+  padding: 10px;
+  padding-top: 7px;
+}
+#contact-button,
+#message-send-btn {
   position: absolute;
-  bottom: 39px;
+  bottom: 15px;
```
Comment (designerofthing) [inline src/App.css:255]: you should prob think about writing it like this:
bottom: 
image ;

**Review by designerofthing — APPROVED** [review]
nice work

**Review by PaulineBA — APPROVED** [review]

File: src/App.css (hunk 1 above)
Comment (jysmys) [reply src/App.css:255]: haha, could u help me?

**Review by kermit-klein — APPROVED** [review]

File: src/components/AboutReQuest.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        >
+          contact
+        </Button>
+      )}
+      {props.statusMessage && (
+        <span id="selected-message">{props.statusMessage}</span>
+      )}
+    </div>
+  );
+};
```
Comment (erikbjoern) [inline src/components/AboutReQuest.jsx:72]: Two components in one? Is this the correct way of doing it? Seems good, I just have never seen it before.

**Review by ViaMarcus — APPROVED** [review]

**Pull request merged**